### PR TITLE
Adding reference to OpenSPP-API

### PR DESCRIPTION
**Pull request by kneckinator**
With this reference, it is possible to install the OpenSPP-API apps in OpenSPP.

### CODE REVIEW

File: odoo/custom/src/repos.yaml [hunk 1]
```diff
@@ -216,7 +229,7 @@ openspp_base:
     openspp: https://github.com/OpenSPP/openspp-base.git
   target: openspp 15.1.1
   merges:
-    - openspp 15.1.1
+    - openspp module_spp_audit_log
```
Comment (jeremi) [inline odoo/custom/src/repos.yaml:232]: Do we want to default on this branch?

File: odoo/custom/src/repos.yaml [hunk 2]
```diff
@@ -53,6 +53,19 @@ openspp_registry:
...
+  defaults:
+    # Shallow repositories ($DEPTH_DEFAULT=1) are faster & thinner
+    # You may need a bigger depth when merging PRs (use $DEPTH_MERGE
+    # for a sane value of 100 commits)
+    depth: $DEPTH_DEFAULT
+  remotes:
+    openspp: https://github.com/openspp/openspp-api.git
+  target: openspp 15.1.1
+  merges:
+    - openspp spp_dci_api_server
```
Comment (jeremi) [inline odoo/custom/src/repos.yaml:66]: Should we do `15.1.1` instead

**Review by jeremi — CHANGES_REQUESTED** [review]

File: odoo/custom/src/repos.yaml (hunk 1 above)
Comment (kneckinator) [reply odoo/custom/src/repos.yaml:232]: Oops. Yes. Thanks for spotting.

File: odoo/custom/src/repos.yaml (hunk 2 above)
Comment (kneckinator) [reply odoo/custom/src/repos.yaml:66]: Valid point. I updated this.

Comment (kneckinator) [reply odoo/custom/src/repos.yaml]: However, there is no `15.1.1` in openspp-api – only `15.0`. How do you suggest that we proceed @jeremi ?

File: odoo/custom/dependencies/pip.txt [hunk 3]
```diff
@@ -23,3 +23,11 @@ pydantic
...
 typing-extensions
 schwifty
+
+# OpenSPP-API dependencies
+bravado_core
+jsonschema
+jwt
+PyLD
+swagger_spec_validator
+cryptography<38
```
Comment (kneckinator) [inline odoo/custom/dependencies/pip.txt:33]: Self-comment: This prevents OpenSPP JWT Rest API Authentication from being installed, as it requires pyOpenSSL 22.0.0 (or higher) to be installed.

@reichie020212 @jeremi we need to resolve this by adjusting the code in `openspp_api` to support the latest version of pyOpenSSL.

Comment (kneckinator) [reply odoo/custom/dependencies/pip.txt:33]: I will remove the pinning of `cryptography` now.

**Comment by kneckinator** [comment]
PR is stale and also no longer relevant. Closing it.

**Pull request closed without merging**